redbiom/admin.py

```diff
@@ -264,14 +264,26 @@ def load_sample_data(table, context, tag=None, redis_protocol=False):
                                           table.metadata(axis='observation'))
     if taxonomy is not None:
         post(context, 'HSET', "state/has-taxonomy/1")
+        hmgetter = redbiom._requests.buffered
+
+        tip_names = {n.name: n for n in taxonomy.tips()}
+        ids_ = hmgetter(tip_names.keys(), None, 'HMGET', context,
```

Comment:
do we really need ids_? can we loop in the for below directly and do hmgetter on each element - not sure if better? Perhaps we will save some loops like this ...

Reply:
It's a generator so it shouldn't matter as it won't get unrolled until the forloop.